### ISSUE TITLE
Boost.Python RuntimeWarnings on `import ledger`

### ISSUE THREAD

**Issue by crowsonkb** [ledger/ledger#1867]
When I run `ledger python -f ledger.dat` and then run `import ledger`, I get a lot of `RuntimeWarning` messages. They look like the following:

```
/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: to-Python converter for boost::posix_time::ptime already registered; second conversion method ignored.
  return f(*args, **kwds)
```

Here is the [entire output](https://gist.github.com/crowsonkb/2524c3982aa89a42b5c987de0dd12cb0).

I compiled Ledger from master: specifically, commit 40d01c3. I am on macOS Catalina (10.15.3). My Python version is 3.7.6, obtained through Homebrew. My Boost.Python version is 1.72.0, also obtained through Homebrew.

**Comment by tbm** [comment]
adding @xnox

**Comment by tbm** [comment]
@xnox any idea what's going on there?

**Comment by q3aiml** [comment]
This should be fixed by #1874
